src/Explorer/OpenActions.ts

```diff
@@ -79,6 +79,14 @@ function openCollectionTab(
           break;
         }
 
+        if (
+          action.tabKind === ActionContracts.TabKind.MongoSchema ||
+          (<any>action).tabKind === ActionContracts.TabKind[ActionContracts.TabKind.MongoSchema]
```

Comment:
What is going on here?

Reply:
Not sure. I was just following the pattern here. I think these are contracts used to programmatically launch a tab (triggered by some window message?). Looks like `tabKind` here can be enum (int) or a string.